package.json

```diff
@@ -58,12 +58,14 @@
     "babel-plugin-module-resolver": "^5.0.0",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.18.3",
+    "cheerio": "^1.0.0-rc.10",
     "cors": "^2.8.4",
     "cron": "^1.8.2",
     "dotenv": "^5.0.1",
     "express": "^4.17.1",
     "express-validator": "^7.0.1",
     "googleapis": "^100.0.0",
+    "jsdom": "^25.0.0",
```

Comment:
Where is jsdom used? Please ignore it if it is peer dependency.

Reply:
It has been removed. Thanks. Please review again.